### ISSUE TITLE
Browser hang after update to 6.6.95

### ISSUE THREAD

**Issue by pdapnz** [Templarian/MaterialDesign-Webfont#84]
Hi!

We are using Material Design Icons through CDN link with `latest` specifier:

```html
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/@mdi/font@latest/css/materialdesignicons.min.css">
```

Project based on VuetifyJs. And after [yesterday update to 6.6.95](https://github.com/Templarian/MaterialDesign-Webfont/releases/tag/v6.6.95) we discovered strange behavior. Browser become hang to some  minutes and not response. After that in Firefox browser become notice "Could not load CSS". Similar situation in Chrome.

Fallback to previous version 6.5.95 solve the problem:

```html
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/@mdi/font@6.5.95/css/materialdesignicons.min.css">
```

**Comment by jendib** [comment]
Same issue here, it took me a while to understand that it came from this font.

**Comment by Warhead5555** [comment]
Took me a while to pinpoint the issue. After blocking the network calls to https://cdn.jsdelivr.net/npm/@mdi/font@latest/css/materialdesignicons.min.css i figured it out. Glad to see others experiencing the same.

**Comment by goyney** [comment]
> Note: Please use the main [MaterialDesign](https://github.com/Templarian/MaterialDesign/issues) repo to report issues. This repo is for distribution of the Webfont files only.

**Comment by DeadActive** [comment]
All of the fonts files (.ttf, .eof, .woff, .woff2) in 6.6.95 hangs up both chrome and firefox on Windows, and there is no issue on Linux based systems. Files seems to be corrupted.

**Comment by Templarian** [comment]
Interesting. The the version of the `webfont` package got bumped as it was very out of date. I might need to revert that change in our `@mdi/font-build` toolchain and republish. Please following the above issue in the main repo.

**Comment by Templarian** [comment]
This was resolved in `6.6.96` released the other week.
